src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends FigureColor {
+    private int side;
+
+    public Square(int side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: " + String.format("%.1f", getArea())
```

Comment:
Use the 'name()' method of the enum constant for getting the String representation instead of 'getColor()'. If 'getColor()' is a method that you implemented, make sure it returns the name of the enum constant.